pom.xml

```diff
@@ -265,6 +265,12 @@
         <version>2.2</version>
       </dependency>
 
+      <dependency>
```

Comment:
Add a whole other plugin as a dependency seems like a bad idea. Why is this needed? Just for the ResourceTransformer?